### PR TITLE
Write to Excel Feature

### PR DESCRIPTION
**Pull request by minneakhmetov**
This PR does not pretend to be merged.
It is suggestion how to implement writing a collection of POJOs to Excel Sheet using only one line of code.
The demo of usage my code is in tests.
Now annotation @ExcelCellName is only supported.
Please, be kind to review and comment.

### CODE REVIEW

**Comment by ozlerhakan** [comment]
Hi @saiple , thank you for contributing to Poiji :) I will review your changes

File: src/main/java/com/poiji/bind/Poiji.java [hunk 1]
```diff
@@ -370,6 +372,93 @@ private static HSSFPropertyStream deserializerPropertyStream(PoijiExcelType exce
...
+                    object = field.get(dto);
+                } catch (IllegalAccessException e) {
+                    throw new PoijiException("Error during processing POJO to Excel", e);
+                }
+                setCellValue(cell, object, field.getType(), options);
+            }
+        }
+    }
+
+    private static void setCellValue(Cell cell, Object value, Class<?> clazz, PoijiOptions options) {
```
Comment (ozlerhakan) [inline src/main/java/com/poiji/bind/Poiji.java:418]: I think we need to move this logic to another class, we have a similar class called https://github.com/ozlerhakan/poiji/blob/master/src/main/java/com/poiji/config/DefaultCasting.java for deserialization.

File: src/main/java/com/poiji/bind/Poiji.java [hunk 2]
```diff
@@ -370,6 +372,93 @@ private static HSSFPropertyStream deserializerPropertyStream(PoijiExcelType exce
...
+
+    public static <T> void toExcel(final Sheet sheet,
+                                   final Collection<T> objects,
+                                   final Class<T> type,
+                                   final PoijiOptions options) {
+        Field[] fields = type.getDeclaredFields();
+        List<Field> headers = new ArrayList<>();
+
+        int rowIndex = 0;
+        Row header = sheet.createRow(rowIndex);
```
Comment (ozlerhakan) [inline src/main/java/com/poiji/bind/Poiji.java:389]: I think, these logics need to move to another class to abstract the core process.

**Review by ozlerhakan — COMMENTED** [review]
I have added my comments on your changes @saiple

File: src/main/java/com/poiji/bind/Poiji.java (hunk 2 above)
Comment (saiple) [reply src/main/java/com/poiji/bind/Poiji.java]: Hi @ozlerhakan! Sorry for long delay. I fixed my code by your review.

File: src/main/java/com/poiji/config/DefaultCasting.java [hunk 3]
```diff
@@ -313,4 +318,54 @@ public List<DefaultCastingError> getErrors() {
...
 
+    /**
+     * @param cell target cell, which should be set value
+     * @param value target value
+     * @param clazz class of value
+     * @param options PoijiOptions
+     */
+    public static void setCellValueByClass(Cell cell, Object value, Class<?> clazz, PoijiOptions options) {
+        Workbook wb = cell.getSheet().getWorkbook();
+        if (clazz.equals(String.class)) {
```
Comment (furoraru) [inline src/main/java/com/poiji/config/DefaultCasting.java:329]: we need something like this if our value is null
`|| value == null`

**Comment by ozlerhakan** [comment]
Hi @minneakhmetov ,

When do you think you're done?

**Comment by ozlerhakan** [comment]
It has been a long while that the PR is getting old without changes. I'm closing it feel free to make changes we can open it up again.

**Pull request closed without merging**